src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -565,6 +571,36 @@ private static final void addConnectorOptions(ConfigDef config) {
     ).define(
         TOPIC_PREFIX_CONFIG,
         Type.STRING,
+        "",
+        new Validator() {
+          @Override
+          public void ensureValid(final String name, final Object value) {
+            if (value == null) {
+              throw new ConfigException(name, value, "Topic prefix must not be null.");
+            }
+
+            String trimmed = ((String) value).trim();
+
+            if (!trimmed.isEmpty()) {
+              char first = trimmed.charAt(0);
+              if (Stream.of('_', '-', '.').collect(Collectors.toSet()).contains(first)) {
+                throw new ConfigException(name, value,
+                    "Topic prefix must not start with '_', '-' or '.'.");
```

Comment:
Source for this?

Reply:
> Source for this?

I don't know why I put the limitation, just removed it.